ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # ChangeLog for githash
 
+## 0.1.0.2
```

Comment:
The version bump should be to 0.1.1.0 following PVP versioning.

Reply:
sorry that I wasn't aware of it, version # is corrected.